platform/src/main/java/org/stellar/anchor/platform/data/JdbcCustodyTransaction.java

```diff
@@ -84,4 +84,8 @@ public class JdbcCustodyTransaction {
   @SerializedName("kind")
   @Column(name = "kind")
   String kind;
+
+  @SerializedName("attempt_count")
+  @Column(name = "attempt_count")
+  int attemptCount;
```

Comment:
What do you think abount `reconciliationAttemptCount`? Because it's not clear which count it is